package.json

```diff
@@ -41,8 +44,11 @@
     "eslint-plugin-import": "2.22.1",
     "istanbul": "0.4.5",
     "jasmine": "3.5.0",
-    "parse": "3.3.1",
+    "parse": "5.3.0",
+    "parse-server": "^7.3.0",
     "parse-server-conformance-tests": "1.0.0",
```

Comment:
Do we still need the conformance tests, if we already have integration tests?

Reply:
I’m not entirely sure if we still need them, but I think they provide a clear view of the configuration landscape and ensure backward compatibility. On the other hand, since we haven't fully tested the integration tests and they don’t cover all input parameters, we might still need the conformance tests to fill in those gaps.

Reply:
I would rather get rid of the conformance repo. Let's try that once we have the tests up and running.

Reply:
See https://github.com/parse-community/parse-server-s3-adapter/pull/224#issuecomment-2429420888

Reply:
Yeah, I think you're right, and I'm a bit late to the party I see you've already removed the conformance tests.